test/check-storage-mountpoints

```diff
@@ -191,6 +212,14 @@ class TestStorageMountPoints(VirtInstallMachineCase, StorageCase):
     @disk_images([("", 15), ("", 10)])
     @nondestructive
     def testMultipleDisks(self):
+        """
+        Description:
+            Test the case when multiple disks are selected.
```

Comment:
The test is also checking boot partition requirement - not sure if we have more specific test for that, but if not it should be perhaps mentioned here.

Reply:
Actually I think the biosboot check should go to the Description as a second line.
`Test the bootloader partition requirement is applied to the option availability`
because it does not logically belong to the current description (multiple disks).
This way we are aware right away that this test is testing also something specific (which would maybe ideally be in a separate, maybe more comprehensive test for the requirements).